core/src/main/java/com/pablisco/result/Result.java

```diff
@@ -17,16 +17,16 @@ public final boolean isSuccessful() {
         );
     }
 
-    public final S getSuccess() {
+    public final S getSuccessOrElse(S alternative) {
```

Comment:
I could make a separate ticket for that. However, the purpose of this class is to represent a result, having an observer pattern on it may be out of the scope. Maybe as an extension, something like `FutureResult`